include/sound/sof/topology.h

```diff
@@ -75,11 +75,22 @@ struct sof_ipc_comp {
 #define SOF_MEM_CAPS_CACHE			(1 << 6) /**< cacheable */
 #define SOF_MEM_CAPS_EXEC			(1 << 7) /**< executable */
 
+#define SOF_BUF_OVERRUN_PERMITTED	(1 << 0) /**< overrun will cause ring
+						   * buffer overwrite, instead
+						   * of XRUN
+						   */
+#define SOF_BUF_UNDERRUN_PERMITTED	(1 << 1) /**< underrun will cause
+						   * readback of 0s, instead of
+						   * XRUN
+						   */
```

Comment:
I see it used just above, so this is optional, but usually `BIT(x)` is preferred